spec/requests/api/v1/participants_spec.rb

```diff
@@ -44,7 +44,9 @@ def path(id = nil)
       resume_api_v1_participant_path(ecf_id: id)
     end
 
-    it_behaves_like "an API update endpoint"
+    it_behaves_like "an API update endpoint" do
+      let(:params) { { data: { attributes: attributes.merge(reason: "test") } } }
```

Comment:
Not sure we need these `params` defined? they should be pulled from the `attributes` further up. Is it worth modifying the `an API update endpoint` shared context to have an explicit test that sends some random data along with the standard attributes to ensure an error isn't returned?